### ISSUE TITLE
Python validation doesn't work for arrays...

### ISSUE THREAD

**Issue by bcorrie** [airr-community/airr-standards#312]
In #287 I added an array to handle capturing an array of file names in the field data_processing_files... 

After a number of contortions as to figuring out what was wrong, it looks like the python validator doesn't handle array fields properly. See the error message ```airr.schema.ValidationError: Internal error: array field data_processing.data_processing_files in schema not handled by validation. File a bug report.```

I suppose this is the bug report... 8-)

@schristley it looks to me like this might not be too hard to fix, but wasn't sure... Can you comment??? The other case where an array of strings exists in the spec is handled as a special case (see schema.py, line 370). 

```
FAIL: test_load_repertoire (tests.test_interface.TestInferface)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/airr-community/airr-standards/lang/python/tests/test_interface.py", line 78, in test_load_repertoire
    data = airr.load_repertoire(self.rep_good, validate=True)
airr.schema.ValidationError: Internal error: array field data_processing.data_processing_files in schema not handled by validation. File a bug report.
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/travis/build/airr-community/airr-standards/lang/python/tests/test_interface.py", line 80, in test_load_repertoire
    self.assertTrue(False, 'load_repertoire(): good data failed')
AssertionError: False is not true : load_repertoire(): good data failed
======================================================================
FAIL: test_repertoire_template (tests.test_interface.TestInferface)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/airr-community/airr-standards/lang/python/tests/test_interface.py", line 52, in test_repertoire_template
    result = airr.schema.RepertoireSchema.validate_object(rep)
airr.schema.ValidationError: Internal error: array field data_processing.data_processing_files in schema not handled by validation. File a bug report.
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/travis/build/airr-community/airr-standards/lang/python/tests/test_interface.py", line 55, in test_repertoire_template
    self.assertTrue(False, 'repertoire_template(): repertoire template failed validation')
AssertionError: False is not true : repertoire_template(): repertoire template failed validation
```

**Comment by javh** [comment]
This is fixed as part of  #287, correct?

**Comment by schristley** [comment]
> This is fixed as part of #287, correct?

Yes, for the python library, the validation routine for repertoires for R hasn't been written yet.

**Comment by schristley** [comment]
Fixed in #287
